docs/source/api/recipes/recipes.ipynb

```diff
@@ -835,6 +835,73 @@
     "matching_ids = [k for k, v in H.degree().items() if v == d]\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    # Get all of the dates
</pre><p>I think this refers to the example below. I would replace this with something like:</p><pre class="ql-syntax" spellcheck="false"># get all the nodes with degree in within 3 and 20
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/524/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
Fixed.